### PR TITLE
create and style calculator

### PR DESCRIPTION
**Pull request by m05e5**
# Math magicians: components

 -   Delete all the boilerplate from CRA (text, images, styles).
 -   Create a directory called components.
 -  Inside components, create a new Calculator.js file.
 -  In Calculator.js, create a React component that matches the design

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,30 +1,15 @@
 /* eslint-disable */
 import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator'
 /* eslint-enable */
 
 function App() {
```
Comment (vichuge) [inline src/App.js:7]: - [ ] The `App.js` component should be class-bassed (In the future for this project you'll see why this is important). Please fix this important issue. 🐶

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+/* eslint-disable linebreak-style */
+function Calculator() {
```
Comment (vichuge) [inline src/components/Calculator.js:2]: - [ ] The `Calculator.js` component should be class-bassed (In the future for this project you'll see why this is important). Please fix this important issue. 🐶

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your PR messages are descriptive ✔️

## Aspects to improve ♻️

- [ ] There are some elements that could be deleted, I invite you to delete them from your project. This will help to have a more professional project.
image

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js [hunk 3]
```diff
@@ -1,30 +1,15 @@
 /* eslint-disable */
-import logo from './logo.svg';
 import './App.css';
+import React from 'react';
+import Calculator from './components/Calculator';
 /* eslint-enable */
 
 function App() {
```
Comment (vichuge) [inline src/App.js:7]: - [ ] It looks like `App.js` is still a function in place of a class-based component, please change this. Good job implementing this change on calculator.js now do it on App.js too. 🐶

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ♻️ ⚠️ ⚠️ 🚦 🚦 

Hi dear student, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

star_wars_almost_there


## Aspects to improve ♻️
_Check the comments under the review._

- [ ] Check the comments under the review. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [OPTIONAL] The readme file could have the [microverse template](https://github.com/microverseinc/readme-template). I forgot to ask this at the last code review, please, I invite you to highly consider to add the template on the readme.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 3 above)
Comment (m05e5) [reply src/App.js:7]: Hello. please this is the second time you are blocking me for something that is not even in the requirement. Please if it is a must to use class based components please put it in the requirements

File: src/components/Calculator.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable */
```
Comment (Dipeshtwis) [inline src/components/Calculator.js:2]: [OPTIONAL]

- Avoid `disabling eslint` feature instead, you can fix these by implementing best practices in your code.

**Review by Dipeshtwis — APPROVED** [review]
# Hi @m05e5, 

### Status: Approved ✔️

:+1: You have done all the required changes asked by the previous reviewer. Well Done.

Your project is complete!  There are some optional changes I've suggested to you to make your project best, other than that nothing else to say.
Congratulations! 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

[Highly Recommended]
- Please make sure to make README professional. This is a general requirement. I will highly recommend following the [microverse template](https://github.com/microverseinc/readme-template).


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Dipesh Kumar_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**